Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -545,6 +777,881 @@ def get_ip_reputation(client: Client, score_calc: DBotScoreCalculator, ip, statu
     )
 
 
+def return_params_of_pagination_or_limit(page: int = None, page_size: int = None, limit: int = None):
+    """
+    Returns request params accroding to page, page_size and limit arguments.
+    Args:
+        page (int): page.
+        page_size (int): page size.
+        limit (int): limit.
+    Returns:
+        params (dict).
+    """
+    params = {}
+    if (page_size and not page) or (not page_size and page):
+        raise DemistoException('Please specify page and page_size')
+    elif page and isinstance(page, int) and isinstance(page_size, int):
+        params['offset'] = (page * page_size) - (page_size)
+        params['limit'] = page_size
+    else:
+        params['limit'] = limit or DEFAULT_LIMIT_PAGE_SIZE
+    return params
+
+
+def header_transformer(header: str) -> str:
+    """
+    Returns a correct header.
+    Args:
+        header (Str): header.
+    Returns:
+        header (Str).
+    """
+    if header == 'notify_me':
+        return 'Is Notify Me'
+    if header == 'modified_ts':
+        return 'Modified At'
+    if header == 'created_ts':
+        return 'Created At'
+    if header == 'email':
+        return 'Submitted By'
+    if header == 'numRejected':
+        return 'Excluded'
+    if header == 'numIndicators':
+        return 'Included'
+    if header == 'approved_by':
+        return 'Reviewed By'
+    if header == 'model_type':
+        return 'Type'
+    return string_to_table_header(header)
+
+
+def list_rule_command(client: Client, rule_id: str = None, limit: str = '50', page: str = None,
+                      page_size: str = None) -> CommandResults:
+    """
+    Returns a list rules.
+    Args:
+        client (Client): Client to perform calls to Anomali ThreatStream service.
+        rule_id (Str):  Unique ID assigned to the rule.
+        limit (Str): The maximum number of results to return.
+        page (Str): The page number of the results to retrieve.
+        page_size (Str):  The maximum number of objects to retrieve per page.
+    Returns:
+        (CommandResults).
+    """
+    params = return_params_of_pagination_or_limit(arg_to_number(page), arg_to_number(page_size), arg_to_number(limit))
+    res = client.list_rule_request(rule_id, params)
+    data = res.get('objects', res)
```

Comment:
Don't return `res` as default. return the empty object - list or dict [] or {}
For all places